### PR TITLE
Record tests for CloudService

### PR DESCRIPTION
**Pull request by Youri970410**
## Description

Record tests for CloudService

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/CloudService/readme.md [hunk 1]
```diff
@@ -171,7 +171,20 @@ directive:
...
+    where: $
+    transform: $ = $.replace('            }\n\n            // base64 for everyone else\n            return new JsonString(System.Convert.ToBase64String(content));','')
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('if (content.EndsWith("=="))','if (!(content.Contains("{") || content.Contains("[")))')
 
+  # - from: source-file-csharp
+  #   where: 
+  #   transform: return new JsonString(System.Text.Encoding.UTF8.GetString(content));
+
```
Comment (wyunchi-ms) [inline src/CloudService/readme.md:187]: I think we can create a issue for `autorest.powershell`.

**Review by wyunchi-ms — APPROVED** [review]

File: src/CloudService/readme.md (hunk 1 above)
Comment (Youri970410) [reply src/CloudService/readme.md:187]: Created and xiaogang fixed it in the latest version of "Autorest". But it was not verified yet.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**